wagtail/images/views/chooser.py

```diff
@@ -156,6 +158,45 @@ def image_chosen(request, image_id):
     )
 
 
+def duplicate_found(request, new_image, duplicates):
+    next_step_url = (
+        "wagtailimages:chooser_select_format"
+        if request.GET.get("select_format")
+        else "wagtailimages:image_chosen"
+    )
+
+    confirm_duplicate_upload_action = reverse(next_step_url, args=(new_image.id,))
+
+    existing_image = duplicates[0]
+    choose_existing_image_url = reverse(next_step_url, args=(existing_image.id,))
+    cancel_duplicate_upload_action = (
```

Comment:
Try using an f-string to assemble the string here - minutely more efficient, and a bit neater!